cmd/create.go

```diff
@@ -175,6 +199,7 @@ func gatherFromUser(config createConfig) (c createConfig, err error) {
 	config.Registry = prompt.ForString("Image registry", config.Registry)
 	config.Namespace = prompt.ForString("Namespace at registry", config.Namespace)
 	config.Language = prompt.ForString("Language of source", config.Language)
```

Comment:
I'm wondering if we need a term like "language runtime" since Quarkus is not a language in and of itself.

Reply:
I like the idea of "Runtime" better than language, because the developer is specifying not just the language of the resultant function, but also the entire context within which it operates.  I will update all references to "runtime".  This works well with the switch back to "Template" from "Context" as well, which was half solving the problem. @lance